DuplicatesFinder_v4/ViewModels/MainViewModel.cs

```diff
@@ -151,7 +153,7 @@ public ICommand OnClickExport
             {
                 return onClickExport ?? (onClickExport = new RelayCommand((r) =>
                 {
-                    Task.Run(() => DuplicatesViewModel.SaveToTxt());
+                    Task.Run(() => DuplicatesViewModel.SaveToTxtAsync());

```

Comment:
Incorrect logic.
You wan to start and not wait async method.
In this case you have two options:
1. Use Task.Run to execute **non-async** method
2. Just call async method without waiting (without call of await)

Reply:
Tnx. Fixed